server/models/ProspectModel.py

```diff
@@ -0,0 +1,23 @@
+from app import db 
+
+prospects_tags = db.Table('prospects_tags',
+    db.Column('prospect_id', db.Integer, db.ForeignKey('prospects.id')),
+    db.Column('tag_id', db.Integer, db.ForeignKey('tags.id'))
+)
+
+class ProspectModel(db.Model):
+    __tablename__ = 'prospects'
+
+    id = db.Column(db.Integer, primary_key = True)
+    email = db.Column(db.String(120), unique = True, nullable = False)
+    name = db.Column(db.String(120))
+    status = db.Column(db.String(120))
+    owner_id = db.Column(db.Integer, db.ForeignKey('users.id'))
+    tags = db.relationship(
+        'TagModel', secondary=prospects_tags, backref='prospects', lazy='select'
+    )
```

Comment:
just to confirm, this creates a many to many relationship?

Reply:
Yep!

Reply:
I tried to see if I could create a many to many relationship without having to create an extra table, but did not find anything. I could have also created an extra class, that being the many to many relationship model, but that is inefficient because you generally want to do that when there are other fields that needs to be in there other than foreign keys. But in that case, all we really need is foreign keys